sklearn_numba_dpex/kmeans/kernels/lloyd_single_step.py

```diff
@@ -301,7 +297,7 @@ def fused_lloyd_single_step(
         weight = sample_weight[sample_idx]
 
         privatization_idx = (
-            sample_idx // preferred_work_group_size_multiple
+            sample_idx // sub_group_size
         ) % n_centroids_private_copies
```

Comment:
Could you please insert an inline comment to explain this?

Also doesn't black makes this fit on one line? Maybe not.

Reply:
I autoformat all my files. doesn't our test pipeline test for conformity to black anyway ?

Reply:
it seems that in some block in some files black just refuses to apply :thinking:

Reply:
about the inline comment, it's already here, i reformated a bit and updated to replace preferred_work_group_size with sub_group_size

Reply:
I suspect an imbalanced problem between `# fmt: off` / `# fmt: on` but I could not spot it here.